src/language/LanguageManager.js

```diff
@@ -243,6 +243,28 @@ define(function (require, exports, module) {
     function getLanguage(id) {
         return _languages[id];
     }
+
+    /**
+     * Resolves a file content to a Language object.
+     * @param {!string} rawText  Text content of the file.
+     *
+     * @return {Language} The language detected based on file content or the fallback language
+     */
+    function getLanguageForContent(rawText) {
+        // Shebang detection for script languages 
+        if (_languages && rawText) {
```

Comment:
`_languages` seemed to be null once for some reason when I was debugging.  I will try testing it again and see.